src/ImageSharp.Web.Providers.AWS/Providers/AWSS3StorageImageProviderOptions.cs

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) Six Labors and contributors.
+// Licensed under the Apache License, Version 2.0.
+
+namespace SixLabors.ImageSharp.Web.Providers
+{
+    /// <summary>
+    /// Configuration options for the <see cref="AWSS3StorageImageProvider"/> provider.
+    /// </summary>
+    public class AWSS3StorageImageProviderOptions
+    {
+        /// <summary>
+        /// Gets or sets the bucket name.
+        /// Must conform to AWS S3 bucket naming guidelines.
+        /// </summary>
+        public string BucketName { get; set; }
```

Comment:
Credentials?

Reply:
I was going to do some reading. AWS has multiple ways of initialising the client. Their recommendation is normally to use their extension methods. I will see how I go. We register a single S3 client outside of all of our code and I was thinking of applying something similar but I will confirm through their doco.